mad_cart.gemspec

```diff
@@ -21,8 +21,8 @@ Gem::Specification.new do |spec|
   spec.add_dependency "faraday"
   spec.add_dependency "faraday_middleware"
   spec.add_dependency "etsy", "0.2.6"
-  spec.add_dependency "money", "6.1.1"
-  spec.add_dependency "monetize", "0.3.0"
+  spec.add_dependency "money", "6.7.1"
+  spec.add_dependency "monetize", "1.4.0"
```

Comment:
@ezuk it's safe to assume that we'd require a version of money and monetize `>6` and `>1` respectively?